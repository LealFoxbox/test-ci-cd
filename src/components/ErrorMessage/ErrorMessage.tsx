import React from 'react';
import { Text } from 'react-native-paper';
import styled from 'styled-components/native';

const ColoredText = styled(Text)`
  font-size: 12px;
  color: #ff0d10;
  margin-bottom: 10px;
`;

const ErrorMessage: React.FC<{}> = ({ children }) => <ColoredText>{children}</ColoredText>;

export default ErrorMessage;
