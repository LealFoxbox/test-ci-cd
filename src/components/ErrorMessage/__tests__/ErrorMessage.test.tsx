import React from 'react';
import { render } from '@testing-library/react-native';

import ErrorMessage from '../ErrorMessage';

describe('ErrorMessage', () => {
  it('should render', () => {
    const result = render(<ErrorMessage />);

    expect(result).toMatchSnapshot();
  });
});
