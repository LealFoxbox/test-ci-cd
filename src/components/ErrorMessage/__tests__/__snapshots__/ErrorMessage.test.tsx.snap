// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage should render 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#000000",
        "fontFamily": "System",
        "fontWeight": "400",
        "textAlign": "left",
      },
      Array [
        Object {
          "color": "#B00020",
          "fontSize": 12,
          "marginBottom": 10,
        },
      ],
    ]
  }
/>
`;
